docs/mine-hnt/validators/validators.mdx

```diff
@@ -0,0 +1,47 @@
+---
+id: validators
+title: Validators (Testnet)
+hide_title: true
+sidebar_label: Validators (Testnet)
+slug: /mine-hnt/validators
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+# Validators (Testnet)
+
+## Overview
+
+Validators are a new entity on the Helium Blockchain that will perform the work of the consensus group including verifying transactions and adding new blocks to the blockchain. 
+
+For more information about validators and to register interest in staking, go [here](https://www.helium.com/stake).
```

Comment:
Do we need this linked here Once TN is live?

Reply:
yes, it's not for testing but if users want to stake still want to capture interest - demand data for staking providers